### ISSUE TITLE
Open and close dropdown menus with keyboard

### ISSUE THREAD

**Issue by interim17** [simularium/simularium-website#593]
## Use Case

Sub feature of keyboard navigation/accessibility, currently when Load Model and Help buttons are focused, they cannot be opened/closed with keyboard

## Acceptance Criteria

Pressing `enter`, `spacebar`, or `down arrow` while button is focused and menu is closed: opens the menu
Pressing `escape` closes the menu, return focus to button.
Hover/mouse click interactions still work as expected

Out of scope:  
- handling/focus management for `tab` and `shift+tab` out of open menu
- managing focus inside the drop down menu
